enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -874,6 +867,25 @@ func ConfigureSecondaryNetwork(ctx context.Context) error {
 	if err := networking.AddRouteIfNotPresent(ctx, route); err != nil {
 		return err
 	}
+
+	return nil
+}
+
+// Configure the secondary network for podman so that traffic from podman will be routed through
```

Comment:
nit: ConfigureIsolation configures...also maybe mention the blackholing

Reply:
done